admin/app/adapters/user.js

```diff
@@ -4,7 +4,7 @@ export default class UserAdapter extends ApplicationAdapter {
   urlForQueryRecord(query) {
     if (query.me) {
       delete query.me;
-      return `${super.urlForQueryRecord(...arguments)}/me`;
+      return `${this.host}/${this.namespace}/admin/admin-members/me`;
     }
```

Comment:
**remarque**
En fait je pense que cette fonction (urlForQueryRecord) n'a plus d'utilité ici puisque le seul endroit où elle était appelée était dans `admin/app/services/current-user.js` au moment du `this.store.queryRecord('user')` qui a été remplacé par un `this.store.queryRecord('admin-member')`